core/tests/puppeteer-acceptance-tests/user-utilities/curriculum-admin-utils.ts

```diff
@@ -0,0 +1,825 @@
+const skillReviewMaterialHeader = 'div.e2e-test-open-concept-card';
+const addSkillButton = 'button.e2e-test-add-skill-button';
+const confirmSkillCreationButton =
+  'button.e2e-test-confirm-skill-creation-button';
+
+const editSkillItemSelector = 'i.e2e-test-skill-item-edit-btn';
+const confirmSkillAssignationButton =
+  'button.e2e-test-skill-assign-subtopic-confirm';
+
+const addDiagnosticTestSkillButton =
+  'button.e2e-test-add-diagnostic-test-skill';
+const diagnosticTestSkillSelector =
+  'select.e2e-test-diagnostic-test-skill-selector';
+
+const saveChangesMessageInput = 'textarea.e2e-test-commit-message-input';
+
+const addStoryButton = 'button.e2e-test-create-story-button';
+const storyTitleField = 'input.e2e-test-new-story-title-field';
+const storyDescriptionField = 'textarea.e2e-test-new-story-description-field';
+const storyUrlFragmentField = 'input.e2e-test-new-story-url-fragment-field';
+const createStoryButton = 'button.e2e-test-confirm-story-creation-button';
+const saveStoryButton = 'button.e2e-test-save-story-button';
+const publishStoryButton = 'button.e2e-test-publish-story-button';
+const storyMetaTagInput = '.e2e-test-story-meta-tag-content-field';
+const unpublishStoryButton = 'button.e2e-test-unpublish-story-button';
+
+const addChapterButton = 'button.e2e-test-add-chapter-button';
+const chapterTitleField = 'input.e2e-test-new-chapter-title-field';
+const chapterExplorationIdField = 'input.e2e-test-chapter-exploration-input';
+const createChapterButton = 'button.e2e-test-confirm-chapter-creation-button';
+
+const explorationMobileSaveMenu = '.e2e-test-mobile-options';
+const explorationMobileSaveButton = '.e2e-test-save-changes-for-small-screens';
+
+const mobileToastMessage = '.e2e-test-toast-message';
+const mobileOptionsSelector = '.e2e-test-mobile-options-base';
+const mobileSaveChangesDropdown = '.e2e-test-mobile-changes-dropdown';
+const mobilePublishButton = '.e2e-test-mobile-publish-button';
+
+const mobileTopicSelector = 'div.e2e-test-mobile-topic-name a';
+const mobileSkillSelector = 'span.e2e-test-mobile-skill-name';
+
+const mobileSaveTopicDropdown =
+  'div.navbar-mobile-options .e2e-test-mobile-save-topic-dropdown';
+const mobileSaveTopicButton =
+  'div.navbar-mobile-options .e2e-test-mobile-save-topic-button';
+const mobilePublishTopicButton =
+  'div.navbar-mobile-options .e2e-test-mobile-publish-topic-button';
+const mobileStoryDropdown = '.e2e-test-story-dropdown';
+const mobileSaveStoryChangesDropdown =
+  'div.navbar-mobile-options .e2e-test-mobile-changes-dropdown';
+const mobileSaveStoryChangesButton =
+  'div.navbar-mobile-options .e2e-test-mobile-save-changes';
+const mobilePublishStoryButton =
+  'div.navbar-mobile-options .e2e-test-mobile-publish-button';
+const mobileAddChapterDropdown = '.e2e-test-mobile-add-chapter';
+
+export class CurriculumAdmin extends BaseUser {
+  /**
+   * Function for navigating to the topic and skills dashboard page.
```

Comment:
You can omit "Function for ...", "Function to ..." -- try to document these similar to what we do in Python, start the docstrings with a simple verb.

Reply:
Done